### ISSUE TITLE
Journal name Theringia (Zoologia) should be Iheringia (Zoologia)

### ISSUE THREAD

**Issue by CecSve** [CatalogueOfLife/data#487]
**Describe the problem**:
A GBIF user noticed an incorrect journal name for the species [Lius hector Napp, 1987](https://www.gbif.org/species/9281878). CoL gets the species information from [Jewel Beetles: The World of Jewel Beetles](https://www.catalogueoflife.org/data/dataset/1190), however, I could not find information on the species from the [original resource](http://coleopsoc.org/buprestidae/WorldCat/genera/Lius.htm) so I am not sure where the spelling error occurred. 

The name should be Iheringia (Zoologia). 

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/3VNZY

**Literature references**:
https://www.scielo.br/journal/isz/about/#about

**Comment by yroskov** [comment]
Indeed, looks like OCR typo. Jewel Beetles checklist is a frozen dataset for us. So, change can be done in the CoL directly (if we are able to do it technically).

@gdower, do you have a possibility to change the name of the journal in the CoL reference as pointed above? Correct journal name is **Iheringia (Zoologia)** (typo in CoL: **T**heringia (Zoologia)}
